### PR TITLE
Created table of contents, corrected spelling and punctuation

### PR DESCRIPTION
**Pull request by OlaPom**
- corrected minor stylistic and spelling errors
- added section numbers
- created a table of contents

### CODE REVIEW

**Comment by amtoine** [comment]
> Issue #79
>
you can use https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue directly in the PR description if you want to link it yourself :wink: 
e.g. `should fix #79` :yum:

File: README.md
Comment (amtoine) [inline README.md:1]: i'd only suggest adding *TOC* links to go back to the TOC from any section header, i think it does make documents a bit easier to navigate :relieved: 
> :bulb: see https://github.com/goatfiles/dotfiles#table-of-content for instance

i do not have a strong preference on this, so feel free to tell me it's not worth the effort :wink:

**Review by amtoine — COMMENTED** [review]
simple and great, looks good to me :ok_hand:

File: README.md
Comment (ctmbl) [inline README.md:31]:
```suggestion
Posts are rendered in Markdown, with the [Github Flavored Markdown, GFM](https://github.com/remarkjs/remark-gfm).
```

File: README.md
Comment (ctmbl) [inline README.md:239]: Shouldn't it be
```suggestion
## 5. Upcoming work
```
?
At least I'm pretty sure about the **work**
the TOC would have to be be updated too if you agree!

File: README.md
Comment (ctmbl) [inline README.md:73]:
```suggestion
Before deploying the application, you need to set the environment variables.  
```
so that it adds a newline

**Review by ctmbl — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! I added a few suggestion I already had in mind :wink:

File: README.md
Comment (ctmbl) [reply README.md:1]: :+1:

File: README.md
Comment (amtoine) [reply README.md:239]: if it's "future work" then it's "upcoming work" :+1:

File: README.md
Comment (amtoine) [reply README.md:73]: should it? :smirk:

Comment (ctmbl) [reply README.md:73]: seems clearer to me if it does!

File: README.md
Comment (OlaPom) [reply README.md:1]: done

File: README.md
Comment (OlaPom) [reply README.md]: done

File: README.md
Comment (ctmbl) [reply README.md]: @OlaPom you did not correct the ~`works`~ --> `work` but maybe it is on purpose and I'm mistaken?

Comment (ctmbl) [reply README.md]: Also I must warn you, this is considered a bad reviewing habit to resolve the thread yourself :confused: generally we prefer to let the one that opened it close it :wink:

Comment (OlaPom) [reply README.md]: Sorry for that! I wanted to clean up, but I need to learn more about GitHub etiquette. :blush: I corrected the heading.

Comment (ctmbl) [reply README.md]: No problem! and tks for the modification 😉

**Review by ctmbl — DISMISSED** [review]
Good to go!

**Comment by amtoine** [comment]
@OlaPom 
that looks good but the CI is not happy :smirk: 
you will have to run the same `prettier` command as in the [CI](https://github.com/iScsc/iscsc.fr/blob/main/.github/workflows/prettier.yml#L18-L22) to fix the formatting :relieved:

**Comment by OlaPom** [comment]
I've been looking for instructions, but I need an explanation. Can this be done with GitHub Actions or do I need to install some tools? (Because if it's the latter, I'm afraid my technical skills may not be enough).

**Comment by amtoine** [comment]
@OlaPom 

> I've been looking for instructions, but I need an explanation.
>
this should be added in the repo :+1: 

> Can this be done with GitHub Actions or do I need to install some tools? (Because if it's the latter, I'm afraid my technical skills may not be enough).
>
yeah we chose not to commit in the name of our contributors, to control what's going on in the PR branches :relieved: 
i'm not a `node` nor JS chad, but here is a command that does the job
```bash
npx prettier --write .
```
> **Note**
> see https://prettier.io/docs/en/install.html

**Comment by ctmbl** [comment]
@OlaPom 
> I've been looking for instructions, but I need an explanation. Can this be done with GitHub Actions or do I need to install some tools? (Because if it's the latter, I'm afraid my technical skills may not be enough).

This can't be done with GitHub Actions, because we don't want automated tools to modify our code without our approbation.
However this shouldn't be a problem for this doesn't require any technical skills.

First you need to [install `npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
Then in a terminal (on Windows: Windows Key + R then type "cmd.exe") run `npm install -g prettier`
Finally at the source folder of this repo run  `prettier --write README.md` 
This should do!
If you still believe you can't do it I will np 😉 

PS: to remove it afterwards (if you want to), run `npm uninstall -g prettier`

**Comment by ctmbl** [comment]
@OlaPom 
Also there are conflicts with the main branch, you should solve it **with a git merge not a git rebase**
Again if you want I can do it 😉

**Comment by ctmbl** [comment]
@OlaPom after your `git merge main` you'll have the recent #99 on your dev branch, so you'll be able to just do `npm run pretty` to prettify the code!

> Note that you'll still need to install `prettify` first

**Comment by OlaPom** [comment]
Thank you for the instructions! Let me try :muscle:

**Comment by ctmbl** [comment]
No problem, looking forward to see your commit!

**Review by atxr — APPROVED** [review]
Thanks for you contribution and for correcting my (stupid) English typos :slightly_smiling_face: 
You did a good job :partying_face: 
We just wait for `prettier` now, and then we will merge your work :1st_place_medal: 
*don't hesitate to reach us if you experience any trouble*

**Comment by OlaPom** [comment]
I installed `prettier`, and it works, but the rule doesn’t. When I do `npm run pretty` in the main folder of the repo, I get this:

> iscsc.fr@0.1.2 pretty

> npx prettier --write ‘**/*.{js,md,yml,css,html}’ ‘!mongodb/**’

> [error] No files matching the pattern were found: “‘**/*.{js,md,yml,css,html}’”.

> [error] No files matching the pattern were found: “‘!mongodb/**’”.

I’m on Windows. I googled this error, and it may have sth to do with the quotes.

I copied the readme file to another folder and was able to format it with `prettier --write`. I tried it on a copy for now, because I thought you’d prefer people to run the script. Let me know if it makes any difference. If not, I’ll format the file and push it. Or let me know if I need to do anything about the rule.

**Comment by atxr** [comment]
@OlaPom I ran prettier on my machine and created a pull request on your branch
You can merge it and it will resolve this PR
Sorry for the disturbances

**Comment by ctmbl** [comment]
Like @atxr I managed to run it on Linux, sorry it doesn't work on Windows, but tks for testing it at least we'll be able to solve it now :smiling_face_with_tear:

**Comment by OlaPom** [comment]
Ok. I think it's done. Thanks for help!

**Comment by amtoine** [comment]
CI did pass :partying_face:

**Review by ctmbl — APPROVED** [review]

**Review by atxr — APPROVED** [review]
Thanks again for contributing! :100:

**Pull request merged**
